### PR TITLE
SW-1489 Add government documents as a genre based upon 008/28

### PR DESCRIPTION
**Pull request by lmcglohon**
@jkeck @aeschylus 
tests start at 375 in GenreTests.java file

### CODE REVIEW

File: stanford-sw/src/edu/stanford/StanfordIndexer.java [hunk 1]
```diff
@@ -592,6 +592,20 @@ MSS MEDIA (e.g. MSS MEDIA 0025), MSS PHOTO (e.g. MSS PHOTO 0463), MSS PRINTS (e.
...
+    /** Based upon SW-1489, if the record if for a certain format (MARC, MRDF,
+     *  MAP, SERIAL, or VM and not SCORE, RECORDING, and MANUSCRIPT) and it has
+     *  something in the 008/28 byte, I’m supposed to give it a genre type of
+     *  government document
+    **/
+    if (cf008 != null && cf008.getData().length() >= 29) {
+      if (!main_formats.contains(Format.MUSIC_SCORE.toString()) &&
+          !main_formats.contains(Format.MUSIC_RECORDING.toString()) &&
+          !main_formats.contains(Format.MANUSCRIPT_ARCHIVE.toString()) &&
+          cf008.find("^.{28}[a-z]"))
```
Comment (jkeck) [inline stanford-sw/src/edu/stanford/StanfordIndexer.java]: Is the character in the 008 byte 28 required to be an alpha character?

File: stanford-sw/src/edu/stanford/StanfordIndexer.java [hunk 2]
```diff
@@ -592,6 +592,20 @@ MSS MEDIA (e.g. MSS MEDIA 0025), MSS PHOTO (e.g. MSS PHOTO 0463), MSS PRINTS (e.
       }
     }
 
+    /** Based upon SW-1489, if the record if for a certain format (MARC, MRDF,
+     *  MAP, SERIAL, or VM and not SCORE, RECORDING, and MANUSCRIPT) and it has
+     *  something in the 008/28 byte, I’m supposed to give it a genre type of
+     *  government document
+    **/
```
Comment (jkeck) [inline stanford-sw/src/edu/stanford/StanfordIndexer.java]: WRT the comment (spelling error I think s/record if/record is).

Also, is the `MARC, MRDF, MAP, SERIAL, or VM` logic inherent in one of the other checks below?

File: stanford-sw/src/edu/stanford/StanfordIndexer.java (hunk 1 above)
Comment (lmcglohon) [inline stanford-sw/src/edu/stanford/StanfordIndexer.java]: From the documentation at LOC I believe it should be an alpha character.

File: stanford-sw/src/edu/stanford/StanfordIndexer.java (hunk 2 above)
Comment (lmcglohon) [inline stanford-sw/src/edu/stanford/StanfordIndexer.java]: I believe all but the SERIAL is included - so I'll add that and replace record if with record is.

**Pull request merged**